inardini -- bqml/05_model_training_pipeline_formalization review

inardini opened the pull request:
This PR is about reviewing `05_model_training_pipeline_formalization`.

inardini committed:
allign function docstring to google format. correct model registration

inardini committed:
fix description

polong-lin committed:
added explaination of pipelines and sample image of pipeline, also fixed readme from xgb to logreg

polong-lin commented:
@inardini I made changes to the markdown to improve explanations of pipelines, as everything else LGTM. Could you PTAL again and then merge if you approve of my added changes? Thanks!

inardini committed:
add agreed misc folder. add image in the notebook

inardini commented:
@polong-lin I changed the name of the image folder according to what we agreed (`misc`). Also I added the pipeline image in the notebook. I will merge it
